src/index.html

```diff
@@ -1,13 +1,77 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link
+      rel="stylesheet"
+      href="./styles/style.css"
+    >
+
+    <link
+      rel="stylesheet"
+      href="./styles/reset.css"
```

Comment:
You don't need head link to connect your reset, normalize, etc files, just import the link in the main file(style.css).